### ISSUE TITLE
Support limiter.wrap

### ISSUE THREAD

**Issue by eshao** [SGrondin/bottleneck#41]
This issue is a suggestion.

Assuming a function that should be throttled like

```
var get = function(id) {
   return new Promise(...)
}
```

It would be nice to be able to do this:
```
var get = limiter.wrap(function (id) {
  return new Promise(...)
}
```

Thus, `limiter.wrap` should return a function. It should pass arguments as received and in the same order to the wrapped function.

**Comment by SGrondin** [comment]
This will be added shortly, stay tuned.

**Comment by SGrondin** [comment]
This has been implemented in master and will be released as part of v2.0. I'll let you know when it's released.
Here's what it looks like:
```javascript
var Bottleneck = require('./lib/bottleneck')
var limiter = new Bottleneck({maxConcurrent: 1, minTime: 500}) // v2 only

var get = function (id, text) {
  return new Promise(function (resolve, reject) {
    return resolve(`<${id}: ${text}>`)
  })
}

var wrapped = limiter.wrap(get)

var t0 = Date.now()
wrapped('abc', 12345)
.then((result) => { console.log(`${Date.now() - t0}ms: DONE ${result}`) })
.catch((err) => { console.log('ERROR!!', err) })

wrapped('def', true)
.then((result) => { console.log(`${Date.now() - t0}ms: DONE ${result}`) })
.catch((err) => { console.log('ERROR!!', err) })

```
```bash
~/dev/bottleneck$ node test.js
6ms: DONE <abc: 12345>
505ms: DONE <def: true>
```

**Comment by SGrondin** [comment]
Version 2 has been released, please give it a try. Your feedback is welcome and appreciated!
